src/DurableTask.SqlServer/SqlOrchestrationService.cs

```diff
@@ -504,14 +505,19 @@ public override async Task CreateTaskOrchestrationAsync(TaskMessage creationMess
             command.Parameters.Add("@InputText", SqlDbType.VarChar).Value = startEvent.Input;
             command.Parameters.Add("@StartTime", SqlDbType.DateTime2).Value = startEvent.ScheduledStartTime;
 
+            if (dedupeStatuses?.Length > 0)
+            {
+                command.Parameters.Add("@DedupeStatuses", SqlDbType.VarChar).Value = string.Join(",", dedupeStatuses);
```

Comment:
What will happen if "Running" is not among the dedupeStatuses passed in? Will this clubber a running orchestration?

Reply:
Correct

Reply:
OK. Will not be a problem with our use, as "Running" is always included. I'm not sure what the Azure Storage backend would do?

Reply:
I just tested the behaviour of the Azure Storage backend. It seems that it will NOT clobber a running instance, even when "Running" is not among the dedupeStatuses. I don't get an exception (but then again, I never do with the Azure Storage backend) but the running orchestration will finish normally and not be restarted. Not so with "Pending" - a Pending orchestration _will_ be replaced with a new one, if Pending is not among the dedupeStatuses.